### PR TITLE
add simple-gallery-js@1.0.3 w/ npm auto-update

### LINKED ISSUE #8593: [Author] [Request] Add simpleGallery.js
**Library name:** simpleGallery.js
**Git repository url:** https://github.com/fccoelho7/simpleGallery.js
**npm package url(optional):** https://www.npmjs.com/package/simple-gallery-js
**License(s):** MIT
**Official homepage:** In progress.
**Wanna say something? Leave message here:**
# Upload, surely, is a hard part in programming, probably you have had problems with that. Why not simplify? No frameworks, just 2kb gzipped!

Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dada0423**
close #8593, cc @fccoelho7

Pull request for issue: #8593
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/fccoelho7/simpleGallery.js
 * Official website (optional, not the repository): https://fccoelho7.github.io/simpleGallery.js/
 * NPM package url (optional):https://www.npmjs.com/package/simple-gallery-js
 * License and its reference:MIT
 * GitHub / Bitbucket popularity (required):
   - Count of stars:67
   - Count of watchers:5
   - Count of forks:7
 * NPM download stats (optional):
   - Downloads in the last day:5
   - Downloads in the last week:7
   - Downloads in the last month:38

# Essential checklist
 * [ ] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by dada0423** [comment]
@pvnr0082t  please help me review this PR. thank you

File: ajax/libs/simpleGallery.js/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "simpleGallery.js",
```
Comment (pvnr0082t) [inline ajax/libs/simpleGallery.js/package.json]: @dada0423 Why not use the same library name as upstream? https://github.com/fccoelho7/simpleGallery.js/blob/master/package.json#L2

**Comment by dada0423** [comment]
@pvnr0082t I have correct it. Please help me review this PR. thank you

**Comment by pvnr0082t** [comment]
@dada0423 You just fixed name. How about other problems I mentioned?

**Comment by dada0423** [comment]
@pvnr0082t I have correct it. Please help me review this PR. thank you

**Comment by pvnr0082t** [comment]
@dada0423 You just fixed one problem each time? How about filling the template completely?

**Comment by pvnr0082t** [comment]
LGTM
@cdnjs/intern2 Please help review this PR, thanks.

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by kennynaoh** [comment]
LGTM

**Comment by dada0423** [comment]
@pvnr0082t   Please help review this PR, thanks

**Pull request merged**